docs/pool-upgrade-how-to.md

```diff
@@ -160,6 +169,8 @@ application version:
        ```bash
        sudo chown $(whoami) $HOME/.dcl/cosmovisor/upgrades/vX.X.X/bin/dcld
        sudo chmod a+x $HOME/.dcl/cosmovisor/upgrades/vX.X.X/bin/dcld
+       ```
+       > **_Note:_** If you use `auto-upgrade` with `manual-update`, then `manual-update` will work and `auto-upgrade` will be ignored.
```

Comment:
Very short and confusing. Please options more explicit:
When `auto-download` is enabled
    a) Binary is put manually to the correct path
       ...
    b) Binary is not there
       ...
Also make sure that propese plan should contain the link and the checksum to the binary according the format specified in cosmovisor docs